.github/workflows/publish-to-npm.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - name: "Upload build artifacts"
         uses: "actions/upload-artifact@v4"
         with:
+          name: "dist"
```

Comment:
This is optional, but do you want to call it something more readable like `clp-ffi-js` or `clp-ffi-js-dist` so that it's obvious when downloaded?

Reply:
if you mean where a developer downloads the artifacts from the job's Summary page, sure

Reply:
Yep